tests/kernel/dsp_sharing/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+# Copyright (c) 2022 Synopsys
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(dsp_sharing)
+
+target_sources(app PRIVATE
+    src/main.c
+    src/load_store.c
+    )
+
+if(CONFIG_ISA_ARCV2)
+zephyr_include_directories(
+      ${ARCMWDT_TOOLCHAIN_PATH}/MetaWare/arc/lib/src/fx/include/
+    )
+target_sources(app PRIVATE src/calculation_arc.c)
```

Comment:
For other architectures this test now will not build as there is no implementation of "void test_calculation(void)", but that's ok for me. At least the test is now prepared for extension by others.